### DISCUSSION TITLE
Xp12 support

### DISCUSSION THREAD

**Discussion by Aviodoc** [mauer/xmidictrl#17] [Ideas]
Hello Marco, any plan to make this plug in compatible with xp12?

**Comment by mauer** [comment]
Hello Nefass,

I'm using XMidiCtrl with X-Plane 11 without any problems. Therefore, I would consider it to be compatible. Did you encounter any problems?

Thanks,
Marco

**Reply by Aviodoc** [reply]
Sorry for my question, I tought that you should make some change to make it compatible with X-Plane 12 but from some rapid tests I can confirm that it is working properly with both the HS CL60 and the Toliss A319.
Thank you very much for your plugin it does improve my immersion a lot,
Gian Marco
